go.mod

```diff
@@ -41,5 +41,6 @@ require (
 replace (
 	github.com/gogo/protobuf => github.com/gogo/protobuf v1.3.2
 	golang.org/x/sys v0.0.0 => github.com/golang/sys v0.0.0-20190813064441-fde4db37ae7a
+	golang.org/x/text => golang.org/x/text v0.3.7
```

Comment:
can we move this to indirect requirement ?

Reply:
Done